### PR TITLE
Use seperate files

### PR DESCRIPTION
**Pull request by Thaodan**
use seperate files rather than using the old global file(s) for handling modules

### CODE REVIEW

File: ubuntu/18.04/install.sh [hunk 1]
```diff
@@ -56,22 +56,20 @@ rmmod vsock
 modprobe hv_sock
 
 # Blacklist the vmw module
-cat /etc/modprobe.d/blacklist.conf | grep vmw_vsock_vmci_transport > /dev/null
-if [ "$?" == "1" ]; then
-    bash -c 'echo "blacklist vmw_vsock_vmci_transport" >> /etc/modprobe.d/blacklist.conf <<EOF
-EOF'
+if [ -e /etc/modprobe.d/blacklist_vmw_vsock_vmci_transport.conf ] ; then
```
Comment (jterry75) [inline ubuntu/18.04/install.sh]: You really want a `! -e ...` right?

File: ubuntu/18.04/install.sh [hunk 2]
```diff
@@ -56,22 +56,20 @@ rmmod vsock
...
 fi
 
-# Ensure hv_sock gets loaded
-cat /etc/modules | grep hv_sock > /dev/null
-if [ "$?" == "1" ]; then
-    bash -c 'echo "hv_sock" >> /etc/modules <<EOF
-EOF'
+
+#Ensure hv_sock gets loaded
+if [ ! -e /etc/modules-load.d/hw_sock.conf ] ; then
```
Comment (jterry75) [inline ubuntu/18.04/install.sh]: Out of curiosity why did you change the file name to be `hw_sock.conf` vs `hv_sock.conf` since the module is `hv`?

**Comment by Thaodan** [comment]
Those were typos, please look again.

**Comment by jterry75** [comment]
Awesome thanks!
